### ISSUE TITLE
Understanding which values are missing from varlist

### ISSUE THREAD

**Issue by oloapinivad** [EC-Earth/ece2cmor3#435]
Hi all,

I am currently able to cmorize with success (i.e. ending the ece2cmor command without error) all variables from `cmip6-data-request-varlist-CMIP-historical-EC-EARTH-AOGCM.json`

However, when I check I can see some of them are missing. I have this bash script that compares variables from the varlist.json and obtained output

````
----------------------------------------------
Variables from varlist vs. Variables CMORIZED!
-------- Table CMIP6 ------------------------
Experiment cj02 ------------------ Year 1850 
----------------------------------------------
3hr -> Theory: 22 Actual: 21
6hrLev -> Theory: 5 Actual: 1
6hrPlev -> Theory: 6 Actual: 4
6hrPlevPt -> Theory: 8 Actual: 8
AERday -> Theory: 1 Actual: 1
AERmonZ -> Theory: 1 Actual: 0
Amon -> Theory: 48 Actual: 44
CF3hr -> Theory: 4 Actual: 2
CFmon -> Theory: 1 Actual: 0
E3hr -> Theory: 4 Actual: 4
E3hrPt -> Theory: 1 Actual: 0
Eday -> Theory: 9 Actual: 9
EdayZ -> Theory: 2 Actual: 0
Emon -> Theory: 10 Actual: 10
LImon -> Theory: 8 Actual: 8
Lmon -> Theory: 6 Actual: 6
Omon -> Theory: 105 Actual: 67
SIday -> Theory: 9 Actual: 9
SImon -> Theory: 28 Actual: 27
day -> Theory: 36 Actual: 36
Oclim -> Theory: 7 Actual: 0
Oday -> Theory: 3 Actual: 3
Ofx -> Theory: 7 Actual: 7
Oyr -> Theory: 31 Actual: 0

TOTAL -> Theory: 362 Actual: 267 i.e. 73.75 % 
Total space occupied by one year of exp cj02 is: 84G
````

Of course, this take into account everything. I am aware that Oclim and Oyr are not available and that we are voluntary dismissing model levels. However, there are a few other errors/warnings (that does not affect the correct workflow of ece2cmor) that I am not 100% sure they should be ok.
For instance

````
2019-04-03 12:50:31 ERROR:ece2cmor3.ifs2cmor: Variable zg has unsupported combination of dimensions [u'latitude', u'plev19', u'time'] and will be skipped.
2019-04-03 12:50:31 ERROR:ece2cmor3.ifs2cmor: Variable ua has unsupported combination of dimensions [u'latitude', u'plev39', u'time'] and will be skipped.
2019-04-03 12:50:31 ERROR:ece2cmor3.ifs2cmor: Variable ta has unsupported combination of dimensions [u'latitude', u'plev39', u'time'] and will be skipped.
`````

or 

````
2019-04-03 13:47:51 ERROR:ece2cmor3.nemo2cmor: Variable tos needed for tos in table 3hr was not found in NEMO output files... skipping task
`````

Another example are the biogeochemical variables included in NEMO varlist, which I am not sure should be there. 

So my question: having the varlist.json for each experiment is great, but is there a way to have also the subset which is correctly cmorized by ece2cmor3? I am asking this - even if I am aware is quite tough thing - because I wonder how each research center will proceed. For example, do KNMI, BSC and SMHI all get the same number of cmorized variables for the DECK runs?

Thanks for any hint - and sorry if this is an already known issue.

**Comment by goord** [comment]
Hi Paolo, thanks for your remarks. You are correct, the whole point of having central varlists is to have a benchmark that everybody should be able to cmorize without errors. 

The 3 atmospheric variables you mention are zonal means, which are not (yet) supported in ece2cmor3.

The thing is... we are _planning_ to support zonal means and climatological variables, so I am not sure whether we should exclude them now...

**Comment by treerink** [comment]
In the `varlists` the preferences for the specified EC-Earth model component are applied and the ignored fields are omitted. The latter makes that the relevant errors pop up clearly and are not lost in an ocean of irrelevant error messages, which in fact is exactly what we want. So I think we do not want to hide the error messages you face now, we want to solve them, so please report them in separate issues if they are different type of problems. The `Oclim` and `Odec` are already reported in #262.

**Comment by oloapinivad** [comment]
For completeness, I report all the error/warnings that I get with the 0.9.9 and historical AOGCM varlist. We can see if this are known or unlisted yet. 

1. Model levels (missing in output, so no problem)
````
2019-04-16 06:54:32 INFO:ece2cmor3.taskloader: Dismissing model level target variable hus in table E3hrPt for component ifs...
2019-04-16 06:54:32 INFO:ece2cmor3.taskloader: Dismissing model level target variable pfull in table Amon for component ifs...
2019-04-16 06:54:32 INFO:ece2cmor3.taskloader: Dismissing model level target variable clw in table Amon for component ifs...
2019-04-16 06:54:32 INFO:ece2cmor3.taskloader: Dismissing model level target variable ua in table 6hrLev for component ifs...
2019-04-16 06:54:32 INFO:ece2cmor3.taskloader: Dismissing model level target variable va in table 6hrLev for component ifs...
2019-04-16 06:54:32 INFO:ece2cmor3.taskloader: Dismissing model level target variable hus in table 6hrLev for component ifs...
2019-04-16 06:54:32 INFO:ece2cmor3.taskloader: Dismissing model level target variable cli in table Amon for component ifs...
2019-04-16 06:54:32 INFO:ece2cmor3.taskloader: Dismissing model level target variable pfull in table CF3hr for component ifs...
2019-04-16 06:54:32 INFO:ece2cmor3.taskloader: Dismissing model level target variable ta in table CF3hr for component ifs...
2019-04-16 06:54:32 INFO:ece2cmor3.taskloader: Dismissing model level target variable ta in table 6hrLev for component ifs...
2019-04-16 06:54:32 INFO:ece2cmor3.taskloader: Dismissing model level target variable cl in table Amon for component ifs...
2019-04-16 06:54:32 INFO:ece2cmor3.taskloader: Dismissing model level target variable ta in table CFmon for component ifs...
````

2. Zonal means (not found any issue where are discussed but I think it is known)
```
2019-04-16 06:54:36 ERROR:ece2cmor3.ifs2cmor: Variable zg has unsupported combination of dimensions [u'latitude', u'plev19', u'time'] and will be skipped.
2019-04-16 06:54:36 ERROR:ece2cmor3.ifs2cmor: Variable ua has unsupported combination of dimensions [u'latitude', u'plev39', u'time'] and will be skipped.
2019-04-16 06:54:36 ERROR:ece2cmor3.ifs2cmor: Variable ta has unsupported combination of dimensions [u'latitude', u'plev39', u'time'] and will be skipped.
```

3.  100 meters wind (not found any issue that discuss it)
````
2019-04-16 06:54:36 ERROR:ece2cmor3.grib_filter: Field missing in the first day of file: code 131.128, level type 105, level 100. Dismissing task ua100m in table 6hrPlev
2019-04-16 06:54:36 ERROR:ece2cmor3.grib_filter: Field missing in the first day of file: code 132.128, level type 105, level 100. Dismissing task va100m in table 6hrPlev
````

4.  Missing NEMO variables (mixed): the `tos` and `sidivvel` at the beginning are a bit worrying, many of the others seems more related to biogeochemistry: I am actually wondering why they are there...
````
2019-04-15 07:45:50 ERROR:ece2cmor3.nemo2cmor: Variable tos needed for tos in table 3hr was not found in NEMO output files... skipping task
2019-04-15 07:45:57 ERROR:ece2cmor3.nemo2cmor: Variable sidivvel needed for sidivvel in table SImon was not found in NEMO output files... skipping task
2019-04-15 07:45:58 ERROR:ece2cmor3.nemo2cmor: Variable dissicos needed for dissicos in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:03 ERROR:ece2cmor3.nemo2cmor: Variable talkos needed for talkos in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:03 ERROR:ece2cmor3.nemo2cmor: Variable phos needed for phos in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:04 ERROR:ece2cmor3.nemo2cmor: Variable o2os needed for o2os in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:04 ERROR:ece2cmor3.nemo2cmor: Variable no3os needed for no3os in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:05 ERROR:ece2cmor3.nemo2cmor: Variable po4os needed for po4os in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:06 ERROR:ece2cmor3.nemo2cmor: Variable dfeos needed for dfeos in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:06 ERROR:ece2cmor3.nemo2cmor: Variable chlos needed for chlos in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:14 ERROR:ece2cmor3.nemo2cmor: Variable intpp needed for intpp in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:15 ERROR:ece2cmor3.nemo2cmor: Variable intppnitrate needed for intppnitrate in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:15 ERROR:ece2cmor3.nemo2cmor: Variable intppdiat needed for intppdiat in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:15 ERROR:ece2cmor3.nemo2cmor: Variable intppcalc needed for intppcalc in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:16 ERROR:ece2cmor3.nemo2cmor: Variable intppmisc needed for intppmisc in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:16 ERROR:ece2cmor3.nemo2cmor: Variable intpbfe needed for intpbfe in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:16 ERROR:ece2cmor3.nemo2cmor: Variable intpbsi needed for intpbsi in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:16 ERROR:ece2cmor3.nemo2cmor: Variable intpcalcite needed for intpcalcite in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:17 ERROR:ece2cmor3.nemo2cmor: Variable epc100 needed for epc100 in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:17 ERROR:ece2cmor3.nemo2cmor: Variable epfe100 needed for epfe100 in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:17 ERROR:ece2cmor3.nemo2cmor: Variable epsi100 needed for epsi100 in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:18 ERROR:ece2cmor3.nemo2cmor: Variable epcalc100 needed for epcalc100 in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:18 ERROR:ece2cmor3.nemo2cmor: Variable intdic needed for intdic in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:18 ERROR:ece2cmor3.nemo2cmor: Variable spco2 needed for spco2 in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:18 ERROR:ece2cmor3.nemo2cmor: Variable dpco2 needed for dpco2 in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:19 ERROR:ece2cmor3.nemo2cmor: Variable fgco2 needed for fgco2 in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:19 ERROR:ece2cmor3.nemo2cmor: Variable fgo2 needed for fgo2 in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:19 ERROR:ece2cmor3.nemo2cmor: Variable froc needed for froc in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:19 ERROR:ece2cmor3.nemo2cmor: Variable intpn2 needed for intpn2 in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:20 ERROR:ece2cmor3.nemo2cmor: Variable fsn needed for fsn in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:20 ERROR:ece2cmor3.nemo2cmor: Variable frn needed for frn in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:21 ERROR:ece2cmor3.nemo2cmor: Variable fsfe needed for fsfe in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:21 ERROR:ece2cmor3.nemo2cmor: Variable o2min needed for o2min in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:21 ERROR:ece2cmor3.nemo2cmor: Variable zo2min needed for zo2min in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:21 ERROR:ece2cmor3.nemo2cmor: Variable zooc needed for zooc in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:22 ERROR:ece2cmor3.nemo2cmor: Variable zmicro needed for zmicro in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:22 ERROR:ece2cmor3.nemo2cmor: Variable talk needed for talk in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:27 ERROR:ece2cmor3.nemo2cmor: Variable ph needed for ph in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable o2 needed for o2 in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable chl needed for chl in table Omon was not found in NEMO output files... skipping task
`````

5. Climatological NEMO reported in #262 

````
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable dissic needed for dissic in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable dissoc needed for dissoc in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable zooc needed for zooc in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable detoc needed for detoc in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable calc needed for calc in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable phydiat needed for phydiat in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable phymisc needed for phymisc in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable zmicro needed for zmicro in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable zmeso needed for zmeso in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable talk needed for talk in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable ph needed for ph in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable o2 needed for o2 in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable no3 needed for no3 in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable nh4 needed for nh4 in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable po4 needed for po4 in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable dfe needed for dfe in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable si needed for si in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable chl needed for chl in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable co3 needed for co3 in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable co3satcalc needed for co3satcalc in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable pp needed for pp in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable pnitrate needed for pnitrate in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable pbfe needed for pbfe in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable pbsi needed for pbsi in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable pcalc needed for pcalc in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable expc needed for expc in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable expn needed for expn in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable expp needed for expp in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable dcalc needed for dcalc in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable graz needed for graz in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:28 ERROR:ece2cmor3.nemo2cmor: Variable fgco2 needed for fgco2 in table Oyr was not found in NEMO output files... skipping task
2019-04-15 07:46:34 ERROR:ece2cmor3.nemo2cmor: Variable difvho needed for difvho in table Oclim was not found in NEMO output files... skipping task
2019-04-15 07:46:34 ERROR:ece2cmor3.nemo2cmor: Variable difvso needed for difvso in table Oclim was not found in NEMO output files... skipping task
2019-04-15 07:46:34 ERROR:ece2cmor3.nemo2cmor: Variable difvtrto needed for difvtrto in table Oclim was not found in NEMO output files... skipping task
2019-04-15 07:46:34 ERROR:ece2cmor3.nemo2cmor: Variable tnpeo needed for tnpeo in table Oclim was not found in NEMO output files... skipping task
2019-04-15 07:46:34 ERROR:ece2cmor3.nemo2cmor: Variable difvmo needed for difvmo in table Oclim was not found in NEMO output files... skipping task
2019-04-15 07:46:34 ERROR:ece2cmor3.nemo2cmor: Variable difvmto needed for difvmto in table Oclim was not found in NEMO output files... skipping task
2019-04-15 07:46:34 ERROR:ece2cmor3.nemo2cmor: Variable dispkevfo needed for dispkevfo in table Oclim was not found in NEMO output files... skipping task
````

It takes about 15min for NEMO with 1 core and 50 min for IFS with 12 cores on CCA

**Comment by treerink** [comment]
Thanks!

**Comment by goord** [comment]
Hi Paolo, can you check whether there exist u and v fields in the grib file with level type 109 (model levels) and level value 100? In that case EC-Earth is writing height-levels as model-levels, I think this may be a bug in the IFS...

**Comment by oloapinivad** [comment]
Uhm, it seems this is the case. Couldn't be that this is a default IFS feature? I am wondering this since I don't know if we are extracting any other variable on a height level...

```
grib_ls -w,dataDate=18510101,paramId=131 -p dataDate,dataTime,paramId,levelType,level ICMGGcj02+185101 

ICMGGcj02+185101
dataDate    dataTime    paramId     levelType   level       
18510101    600         131         ml          100        
18510101    1200        131         ml          100        
18510101    1800        131         ml          100        
````

**Comment by treerink** [comment]
Concerning the performance: at the KNMI HPC for IFS with 1 node (28 cores) it takes about 23 minutes. Seems in agreement with 50 min using 12 cores by @oloapinivad, who's performance is 7% better then ours at KNMI taking this rough numbers.

**Comment by oloapinivad** [comment]
Could we agree on how many variables can be obtained with ece2cmor3 1.0.0 and in the different experiments, at least in the most standard ones? 

_for historical_
Using `branches/development/2019/r6705-control-output-files/runtime/classic/ctrl/cmip6-output-control-files/CMIP/EC-EARTH-AOGCM/cmip6-experiment-CMIP-historical/cmip6-data-request-varlist-CMIP-historical-EC-EARTH-AOGCM.json` and ece2cmor3 1.0.0 I obtain **265 variables.** 

Has anybody else tried the same setup?
Thanks

**Comment by treerink** [comment]
So most of the skipped variables under point 4 are PISCES variables (see #486), except from `tos` which has been solved in the mean time (see #477) and  the following list which for which I didn't found a skipped message in my latest `test-all-nemo-mip-variables` test:
```
2019-04-15 07:45:57 ERROR:ece2cmor3.nemo2cmor: Variable  sidivvel needed for sidivvel in table SImon was not found in NEMO output files... skipping task
2019-04-15 07:46:06 ERROR:ece2cmor3.nemo2cmor: Variable  dfeos needed for dfeos in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:15 ERROR:ece2cmor3.nemo2cmor: Variable  intppnitrate needed for intppnitrate in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:15 ERROR:ece2cmor3.nemo2cmor: Variable  intppdiat needed for intppdiat in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:15 ERROR:ece2cmor3.nemo2cmor: Variable  intppcalc needed for intppcalc in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:16 ERROR:ece2cmor3.nemo2cmor: Variable  intppmisc needed for intppmisc in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:16 ERROR:ece2cmor3.nemo2cmor: Variable  intpbfe needed for intpbfe in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:16 ERROR:ece2cmor3.nemo2cmor: Variable  intpbsi needed for intpbsi in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:16 ERROR:ece2cmor3.nemo2cmor: Variable  intpcalcite needed for intpcalcite in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:17 ERROR:ece2cmor3.nemo2cmor: Variable  epfe100 needed for epfe100 in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:18 ERROR:ece2cmor3.nemo2cmor: Variable  intdic needed for intdic in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:19 ERROR:ece2cmor3.nemo2cmor: Variable  intpn2 needed for intpn2 in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:20 ERROR:ece2cmor3.nemo2cmor: Variable  fsn needed for fsn in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:20 ERROR:ece2cmor3.nemo2cmor: Variable  frn needed for frn in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:21 ERROR:ece2cmor3.nemo2cmor: Variable  fsfe needed for fsfe in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:21 ERROR:ece2cmor3.nemo2cmor: Variable  zo2min needed for zo2min in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:21 ERROR:ece2cmor3.nemo2cmor: Variable  zooc needed for zooc in table Omon was not found in NEMO output files... skipping task
2019-04-15 07:46:22 ERROR:ece2cmor3.nemo2cmor: Variable  zmicro needed for zmicro in table Omon was not found in NEMO output files... skipping task
```
I guess these are due to an older version (set)?

**Comment by treerink** [comment]
I am closing this issue after #486 is solved. Issue #490 will take over for the current version, note that as long we don't have a test-all-ifs test, we have to report for IFS variables separately there for piControl, historical and scenario etc. Currently piControl has been reported in #490 and historical and scenario have to be added.
